wsmatrix@martin.zurowietz.de/workspacePopup/workspaceSwitcherPopup.js

```diff
@@ -96,7 +96,14 @@ class WorkspaceSwitcherPopup extends SwitcherPopup.SwitcherPopup {
         }
 
         if (this._popupTimeout > 0 && !this._toggle) {
-            this._noModsTimeoutId = GLib.timeout_add(GLib.PRIORITY_DEFAULT, this._popupTimeout + 150, this._finish.bind(this));
+            this._noModsTimeoutId = GLib.timeout_add(
+                GLib.PRIORITY_DEFAULT,
+                this._popupTimeout,
```

Comment:
Was the `+ 150` removed on purpose?

```suggestion
                this._popupTimeout + 150,
```

Reply:
Yes, it added an extra delay to wait for popup.
We don't need it now as there is no delay anymore. This was discussed somewhere but I can't find it.